fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/rest/lookup/helper/FoDLookupHelper.java

```diff
@@ -0,0 +1,145 @@
+/*******************************************************************************
+ * Copyright 2021, 2023 Open Text.
+ *
+ * The only warranties for products and services of Open Text
+ * and its affiliates and licensors ("Open Text") are as may
+ * be set forth in the express warranty statements accompanying
+ * such products and services. Nothing herein should be construed
+ * as constituting an additional warranty. Open Text shall not be
+ * liable for technical or editorial errors or omissions contained
+ * herein. The information contained herein is subject to change
+ * without notice.
+ *******************************************************************************/
+package com.fortify.cli.fod.rest.lookup.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fortify.cli.common.output.transform.fields.RenameFieldsTransformer;
+import com.fortify.cli.fod._common.rest.FoDUrls;
+import com.fortify.cli.fod.rest.lookup.cli.mixin.FoDLookupTypeOptions;
+import kong.unirest.GetRequest;
+import kong.unirest.UnirestInstance;
+import lombok.Getter;
+
+import javax.validation.ValidationException;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FoDLookupHelper {
+    @Getter
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final String[] lookupItems = new String[]{
```

Comment:
What is this String array used for? It doesn't seem to be referenced in any of the changes in this PR? If not used, this should be removed, otherwise maybe better to use an enum?